consoles/VNC.pm

```diff
@@ -122,9 +122,7 @@ my @encodings = (
     },
 );
 
-sub login {
-    my ($self, $connect_timeout) = @_;
-    $connect_timeout //= 10;
+sub login ($self, $connect_timeout = 10) {
```

Comment:
This will break. Setting a default via signatures is *not* equivalent to the current default initialization. We call the function like this:

```
$self->{vnc}->login($args->{connect_timeout});
```

If `$args->{connect_timeout}` is `undef` we don't get the default value with your change.

Reply:
So what do you suggest? Does

```suggestion
sub login ($self, $connect_timeout = undef) {
    $connect_timeout //= 10;
```

work?

Reply:
Yes, this should work.

Reply:
ok, updated the change accordingly